pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go

```diff
@@ -512,10 +511,7 @@ func getMaxEBSVolume(nodeInstanceType string) int {
 	return volumeutil.DefaultMaxEBSVolumes
 }
 
-// getCSINodeListerIfEnabled returns the CSINode lister or nil if the feature is disabled
+// getCSINodeListerIfEnabled returns the CSINode lister
```

Comment:
```suggestion
// getCSINodeLister returns the CSINode lister
```

Reply:
Thank you, addressed in https://github.com/kubernetes/kubernetes/compare/78b49ee8024d99ae5a849cabac7b05a68a4d4e87..e6baa0d43c4c8ad5ca894f1a7327776915d87b22.